src/Kalambury.js

```diff
@@ -80,28 +82,37 @@ function Forfeit(G, ctx) {
   ctx.events.endTurn();
 }
 
-function Ping(G, ctx) {
+function Ping(G, { playerID }, playerData) {
   G.remainingSeconds = 120 - Math.floor((new Date() - G.secret.startTime) / 1000);
+  updatePlayersData(G, playerID, playerData);
```

Comment:
Maybe we shouldn't start game until there is more than one person? Because now, while waiting for other to joint you'll run out of time and lose points 😃

Reply:
Or we can make game require manual "start". This should be easy to setup with some additional "phase"

Reply:
But it can be done as a separate task